common/lib/xmodule/xmodule/course_module.py

```diff
@@ -390,44 +390,46 @@ class CourseFields(object):
         scope=Scope.settings,
         default=[
             {
-                "short_description": _("Getting Started With Studio"),
+                "short_description": _("Course Introduction"),
                 "items": [
                     {
-                        "short_description": _("Add Course Team Members"),
+                        "short_description": _("Welcome Message"),
                         "long_description": _(
-                            "Grant your collaborators permission to edit your course so you can work together."
+                            "Provide a welcome message on course info page at the beginning of the course."
                         ),
                         "is_checked": False,
-                        "action_url": "ManageUsers",
-                        "action_text": _("Edit Course Team"),
+                        "action_url": "CourseUpdates",
+                        "action_text": _("View Course Updates"),
                         "action_external": False,
                     },
                     {
-                        "short_description": _("Set Important Dates for Your Course"),
+                        "short_description": _("Direct learners to edX DemoX"),
                         "long_description": _(
-                            "Establish your course's student enrollment and launch dates on the Schedule and Details "
-                            "page."
+                            "Provide a link to edX DemoX course (an introduction to the edX platform for new learners) in the "
+                            "first week's course content."
                         ),
                         "is_checked": False,
-                        "action_url": "SettingsDetails",
-                        "action_text": _("Edit Course Details &amp; Schedule"),
-                        "action_external": False,
+                        "action_url": "https://www.edx.org/course/demox-edx-demox-1",
+                        "action_text": "Go to edX DemoX",
+                        "action_external": True,
                     },
                     {
-                        "short_description": _("Draft Your Course's Grading Policy"),
+                        "short_description": _("Prompt learners to get started on course"),
                         "long_description": _(
-                            "Set up your assignment types and grading policy even if you haven't created all your "
-                            "assignments."
+                            "Provide a prompt to the learner on the course info page on how to get started with the course "
+                            "(e.g. \"To get started, click on the \"Courseware\" tab at the top of the page.\")"
```

Comment:
Guide learners to use the Courseware tab by adding a prompt to the Course Info page. For example, \"To get started, click \"Courseware\" at the top of the page.\"